stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessagesViewModelFactory.kt

```diff
@@ -32,7 +32,7 @@ public class MessagesViewModelFactory(
     private val chatClient: ChatClient = ChatClient.instance(),
     private val chatDomain: ChatDomain = ChatDomain.instance(),
     private val enforceUniqueReactions: Boolean = true,
-    private val messageLimit: Int = 30,
+    private val messageLimit: Int = MessageListViewModel.DEFAULT_MESSAGE_LIMIT,
```

Comment:
We also have `messageLimit: Int = 30` in `MessagesScreen`

Reply:
Will update there too - nice catch!